float64[] angleLists
float64[] timeLists
---
bool result